src/main/java/com/nicoislost/Zooms.java

```diff
@@ -0,0 +1,92 @@
+package com.nicoislost;
+
+import com.nicoislost.inputs.KeyBinds;
+import net.minecraft.client.option.KeyBinding;
+
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+/**
+ * An enum for the different zoom settings
+ */
+public enum Zooms {
+
+    ZOOM_1(
+            "Zoom 1",
+            ZoomOMatic.CONFIG::zoom1,
+            ZoomOMatic.CONFIG::zoom1,
+            ZoomOMatic.CONFIG::zoom1SmoothCamera,
+            ZoomOMatic.CONFIG::zoom1SmoothCamera,
+            KeyBinds.KEY_BINDING_1
+    ),
+    ZOOM_2(
+            "Zoom 2",
+            ZoomOMatic.CONFIG::zoom2,
+            ZoomOMatic.CONFIG::zoom2,
+            ZoomOMatic.CONFIG::zoom2SmoothCamera,
+            ZoomOMatic.CONFIG::zoom2SmoothCamera,
+            KeyBinds.KEY_BINDING_2
+    ),
+    ZOOM_3(
+            "Zoom 3",
+            ZoomOMatic.CONFIG::zoom3,
+            ZoomOMatic.CONFIG::zoom3,
+            ZoomOMatic.CONFIG::zoom3SmoothCamera,
+            ZoomOMatic.CONFIG::zoom3SmoothCamera,
+            KeyBinds.KEY_BINDING_3
+    ),
+
```

Comment:
I like this idea - I was trying to figure out away to link the key binding to the config - although I'd bind the config to the key binding so, given a key binding, you could get the config and then you'd only have a single concept to track - but that's me

Reply:
Tbh thats kind of a owo-config thing, if i'd write the config myself it'd be more dynamic.

Reply:
It's been nearly 7 years since I did any "serious" Java work, so been able to pass methods as parameters like this is somewhat new to me ... me like 👍